_data/projects/Animations.yml

```diff
@@ -0,0 +1,12 @@
+name: Animations
+desc: An open source collection of animations, from loaders, to menus, and scroll transitions, and anything in between
+site: https://animations.gq
```

Comment:
@UsmanAhmadSaeed https://animations.gq/ 
Not working, doesn't seem to be a markdown file too